library/default-synths-extra.scd

```diff
@@ -481,10 +481,10 @@ SynthDef(\superwavemechanics, {|out, pan, freq, sustain, accelerate, detune=0, v
 	var env, sound, i, r, d, bwr, sz;
 	freq = freq * XLine.ar(1, exp(accelerate), sustain);
 	env = EnvGen.ar(Env.sine(1.0), timeScale:sustain, doneAction:2);
-	i = Array.fill(8, {TIRand.kr(0, 3, Impulse.kr(8))});
+	i = Array.fill(8, {TIRand.kr(0, 3, Impulse.kr(sustain/8))});
```

Comment:
because the 8 `Impulse`Ugens are all the same, you can just have one of them outside the Array and use it as an input for all.

`i = TIRand.kr(0, 3, Impulse.kr(sustain/8).dup(8)))` will expand and do the same.